### PR TITLE
New extension: Simplelogin.io

### PR DESCRIPTION
**Pull request by adrian-schnell**
## Description

Add new SimpleLogin extension for raycast.
With SImpleLogin you can receive and send emails anonymously and protect the inbox against spam and phishing.

## Screencast

https://user-images.githubusercontent.com/322735/211196975-b53bdcbd-abb5-4f3b-8fd5-e78c4ff5fb72.mp4

## Screenshots
simplelogin-1
simplelogin-2

## for testing

1. register on https://simplelogin.io/
2. head over to https://app.simplelogin.io/dashboard/api_key
3. enter API key in the extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/simplelogin/src/list_alias.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                      )}
+                    </>
+                    <Action
+                      title={alias.enabled ? "Disable Alias" : "enable Alias"}
+                      onAction={() => toggleAliasStatePrompt(alias, !alias.enabled)}
+                      icon={!alias.enabled ? Icon.Eye : Icon.EyeDisabled}
+                      shortcut={{ modifiers: ["cmd"], key: "t" }}
+                    />
+                    <Action
+                      title="Delete alias"
```
Comment (pernielsentikaer) [inline extensions/simplelogin/src/list_alias.tsx:186]:
```suggestion
                      title="Delete Alias",
                      style={Action.Style.Destructive},
```

File: extensions/simplelogin/src/list_alias.tsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                          title="Unpinn alias"
+                          onAction={() => updatePinnedStatus(alias, false)}
+                          icon={Icon.PinDisabled}
+                        />
+                      ) : (
+                        <Action title="Pinn alias" onAction={() => updatePinnedStatus(alias, true)} icon={Icon.Pin} />
+                      )}
+                    </>
+                    <Action
+                      title={alias.enabled ? "Disable Alias" : "enable Alias"}
```
Comment (pernielsentikaer) [inline extensions/simplelogin/src/list_alias.tsx:180]:
```suggestion
                      title={alias.enabled ? "Disable Alias" : "Enable Alias"}
```

File: extensions/simplelogin/src/list_alias.tsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                    }
+                  />
+                }
+                actions={
+                  <ActionPanel>
+                    <Action.CopyToClipboard content={alias.email} shortcut={{ modifiers: ["cmd"], key: "." }} />
+                    <>
+                      {alias.pinned ? (
+                        <Action
+                          title="Unpinn alias"
```
Comment (pernielsentikaer) [inline extensions/simplelogin/src/list_alias.tsx:171]:
```suggestion
                          title="Unpin Alias"
```

File: extensions/simplelogin/src/list_alias.tsx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                          onAction={() => updatePinnedStatus(alias, false)}
+                          icon={Icon.PinDisabled}
+                        />
+                      ) : (
+                        <Action title="Pinn alias" onAction={() => updatePinnedStatus(alias, true)} icon={Icon.Pin} />
```
Comment (pernielsentikaer) [inline extensions/simplelogin/src/list_alias.tsx:176]:
```suggestion
                        <Action title="Pin Alias" onAction={() => updatePinnedStatus(alias, true)} icon={Icon.Pin} />
```

File: extensions/simplelogin/src/list_alias.tsx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                          title="Mails blocked"
+                          text={"" + alias.nb_block}
+                          icon={Icon.Eraser}
+                        />
+                        <List.Item.Detail.Metadata.Separator />
+                        <List.Item.Detail.Metadata.Label title="Contained in mailbox" />
+                        <>
+                          {alias.mailboxes.length > 0 &&
+                            alias.mailboxes.map((mailbox) => (
+                              <List.Item.Detail.Metadata.Label title="E-Mail" text={mailbox.email} />
```
Comment (pernielsentikaer) [inline extensions/simplelogin/src/list_alias.tsx:158]:
```suggestion
                              <List.Item.Detail.Metadata.Label title="E-mail" text={mailbox.email} />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* After creating an alias in `Create Alias`, it would be nice to popToRoot or reset the form
* When running `Create Random Alias` it would be nice with a loading spinner since there might be a delay before there is a response ready
* `List Aliases` should use isLoading so empty view not is visible before it's needed. 
* Maybe make a copy of the extension icon in 128x128 or so to use instead of the cat 😊
 
I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by adrian-schnell** [comment]
@pernielsentikaer you can take a look again  ;)

**Comment by sxn** [comment]
hi @adrian-schnell! 👋 hope it's alright, I've refactored the list command a bit to avoid the initial empty state flickering. Mind having a look if everything still works as expected? 😸

**Comment by adrian-schnell** [comment]
thank you 🙏 @sxn

**Comment by pernielsentikaer** [comment]
Let's merge it! 🔥

**Pull request merged**